oomph-plugin-notes.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+Plugin Name: Oomph Plugin Notes
```

Comment:
I think GH highlights this funny (as does Sublime) because when you start a comment with two asterisks, it is expected that every line begin with an asterisk; if you start the comment with only one asterisk then it doesn't make a difference. Not critical...